optuna/pruners/_percentile.py

```diff
@@ -129,6 +130,10 @@ def objective(trial):
             Interval in number of steps between the pruning checks, offset by the warmup steps.
             If no value has been reported at the time of a pruning check, that particular check
             will be postponed until a value is reported. Value must be at least 1.
+        n_min_trials:
+            Minimum number of trials to perform pruning at each step. If the number of reported
+            intermediate values in a trial is less than ``n_min_trials`` at a step,
+            the trial is not pruned at this step.
```

Comment:
```suggestion
            Minimum number of reported trial results at a step to judge whether to prune.
            If the number of reported intermediate values from all trials at the current step
            is less than ``n_min_trials``, the trial will not be pruned. This can be used to ensure
            that a minimum number of trials are run to completion without being pruned.
```